### DISCUSSION TITLE
Images

### DISCUSSION THREAD

**Discussion by ciberelu** [Launch-X-Latam/MisionFrontEnd#46] [Q&A]
Hola viajeros! A ustedes les aparecen las imágenes en la teoría del curso ?

**Comment by CarloAyala** [comment]
Si compañero
<img width="300" alt="image" src="https://user-images.githubusercontent.com/114269986/192657665-7a8a5ddd-949e-46ca-9f50-ad65f0eadc1c.png">

Quizá es tu navegador, intenta probar con otro, intenta vaciando el cache de la pagina, 
1. Clic derecho en cualquier parte de la pagina
2. Seleccionas inspeccionar
3. Presionando la tecla Shift das click derecho en el botón de refresh
4. Seleccionas la ultima opción de vaciar cache

<img width="300" alt="Sin título" src="https://user-images.githubusercontent.com/114269986/192658406-4d0974ae-84b3-4a1c-b296-b4c9afe75add.png">

**Reply by Angosk** [reply]
¿Intentaste en otro explorador? Yo uso chrome con dark reader y se ven bien.

**Comment by Leonardo-Ismet** [comment]
Si es en la app no aparecen, en ese caso es preferible abrirlo en navegador.
